modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/AbstractPlannerTest.java

```diff
@@ -499,8 +499,6 @@ protected <T extends RelNode> void assertPlan(
         String planString = RelOptUtil.dumpPlan("", plan, SqlExplainFormat.TEXT, DEFAULT_EXPLAIN_LEVEL);
         log.info("statement: {}\n{}", sql, planString);
 
-        checkSplitAndSerialization(plan, schemas);
```

Comment:
I would rather introduce an option to disable. this check with a default value that tells to always check.

Reply:
planner tests shouldn't check serialization

Reply:
How are going to ensure that a plan can be sent to another node?

Reply:
KeyValue plans are not serialisable, but it does not mean that all serialisation should be disabled.

Reply:
This check greatly simplifies detection bugs like this one https://issues.apache.org/jira/browse/IGNITE-19858

Reply:
partially covered by our integration tests. If we need wider cover it should be separated tests, not as part of planner tests

Reply:
reworked a bit